src/sqlite/plugins.ts

```diff
@@ -0,0 +1,73 @@
+import { startPlugin, stopPlugin } from 'services/pluginService';
+import { getAsync, runAsync } from 'sqlite';
+
+export const ENABLE_PLUGIN = 'systemctl enable ';
+export const START_PLUGIN = 'systemctl start ';
+export const DISABLE_PLUGIN = 'systemctl disable ';
+export const STOP_PLUGIN = 'systemctl stop ';
```

Comment:
We can move those ones to service itself,
the commands not related much to sqlite!